saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java

```diff
@@ -44,7 +44,7 @@
  *	String registrationId = "simplesamlphp";
  *
  * 	String relyingPartyEntityId = "{baseUrl}/saml2/service-provider-metadata/{registrationId}";
- *	String assertionConsumerServiceLocation = "{baseUrl}/login/saml2/sso/{registrationId}";
+ *	String assertingConsumerServiceLocation = "{baseUrl}/login/saml2/sso/{registrationId}";
```

Comment:
Thank you for the typo cleanups. Will you please place these in a separate commit?